validation/tm-json-schema-validation.json

```diff
@@ -1049,6 +1049,37 @@
         }
       }
     },
+    "autoSecurityScheme": {
+      "type": "object",
+      "properties": {
+        "@type": {
+          "$ref": "#/definitions/type_declaration"
+        },
+        "description": {
+          "$ref": "#/definitions/description"
+        },
+        "descriptions": {
+          "$ref": "#/definitions/descriptions"
+        },
+        "proxy": {
+          "$ref": "#/definitions/anyUri"
+        },
+        "scheme": {
+          "type": "string",
+          "anyOf": [
+            {
+              "enum": [
+                "auto"
+              ]
+            },
+            {
```

Comment:
not quite sure why this is here...

Reply:
I regenerated the TM schema, therefore it has been integrated alongside the other SecuritySchemes.